src/hooks/protections/receive_net_message.cpp

```diff
@@ -2,16 +2,16 @@
 #include "backend/context/chat_command_context.hpp"
 #include "backend/player_command.hpp"
 #include "core/data/packet_types.hpp"
+#include "gta/enums.hpp"
 #include "gta/net_game_event.hpp"
 #include "gta_util.hpp"
 #include "hooking/hooking.hpp"
 #include "lua/lua_manager.hpp"
 #include "natives.hpp"
 #include "script/scriptIdBase.hpp"
 #include "services/players/player_service.hpp"
-#include "util/session.hpp"
 #include "util/chat.hpp"
-#include "gta/enums.hpp"
+#include "util/session.hpp"
```

Comment:
Revert these unnecessary changes to the header.

Reply:
Serious question: What exactly is the problem with this and similar changes (i.e. whitespaces)? Merging problems / more potential work?

Reply:
These were made by the formatter, but i will revert

Reply:
> Serious question: What exactly is the problem with this and similar changes (i.e. whitespaces)? Merging problems / more potential work?

It's just unnecessary.

Reply:
The reordering of includes is fine as that's just a formatter thing to do.